code/espurna/config/general.h

```diff
@@ -736,11 +801,6 @@ PROGMEM const char* const custom_reset_string[] = {
 #define DOMOTICZ_SUPPORT        MQTT_SUPPORT    // Build with domoticz (if MQTT) support (1.72Kb)
```

Comment:
Would this make sense as 0? dependency already checks for this

Reply:
And on second thought - this could go to the same preset as ESPURNA_CORE - ESPURNA_DEFAULT?